src/Models/Test/ModelInstantiatorTest.php

```diff
@@ -422,6 +422,24 @@ public function testConvertPropertyOnSave()
         );
     }
 
+    public function testDontConvertNullPropertyOnSave()
+    {
+        // Privacy uses the yes/no converter, but it's null so it shouldn't be
+        // converted.
+        $data_object = new ModelInstantiatorTestObject();
+        $data_object->privacy = null;
```

Comment:
You mentioned `DATETIME` columns. Do we care about testing that column type specifically?